### PR TITLE
Adding Maintenance RP .NET SDK, ARM Swagger PR https://github.com/Azure/azure-rest-api-specs/pull/6469

### PR DESCRIPTION
**Pull request by abkmr**
Adding Maintenance RP .NET SDK
Approved by ARM

### CODE REVIEW

**Comment by isra-fel** [comment]
Hi @abkmr, the SDK should be placed in `/sdk/{moduleinlowercase}`. I also notice that you missed the metadata file. Maybe you didn't follow [these steps](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-directory-structure-and-sdk-generation) when generating? 🤔

**Comment by abkmr** [comment]
> Hi @abkmr, the SDK should be placed in `/sdk/{moduleinlowercase}`. I also notice that you missed the metadata file. Maybe you didn't follow [these steps](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-directory-structure-and-sdk-generation) when generating? 🤔
> 
> Besides, could you share the link to swagger review PR(s), thank

I had followed that but did not remember to add metadata file. PR for swagger is https://github.com/Azure/azure-rest-api-specs/pull/6469. Updated the casing for rp. thanks

File: src/SDKs/maintenance/Management.Maintenance/Microsoft.Azure.Management.Maintenance.csproj [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.Maintenance</PackageId>
+    <Description></Description>
+    <Version>1.0.0</Version>
+    <AssemblyName>Microsoft.Azure.Management.Maintenance</AssemblyName>
+    <PackageTags>management;maintenance;</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
+
```
Comment (isra-fel) [inline src/SDKs/maintenance/Management.Maintenance/Microsoft.Azure.Management.Maintenance.csproj]: Please fill in the relese notes

File: src/SDKs/_metadata/maintenance_resource-manager.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: latest
```
Comment (isra-fel) [inline src/SDKs/_metadata/maintenance_resource-manager.txt]: This file is in the wrong path, it should be in `eng\mgmt\mgmtmetadata\`. Could you run

> msbuild eng/mgmt.proj /t:Util /p:UtilityName=InstallPsModules

then re-run `generate.ps1`

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @abkmr , please

- check in `AssemblyInfo.cs`
- write tests for SDK code

and please see my other inline comments. Thanks!

File: src/SDKs/_metadata/maintenance_resource-manager.txt (hunk 2 above)
Comment (abkmr) [reply src/SDKs/_metadata/maintenance_resource-manager.txt]: Looks like my repo fork was done when all RPs had Caps folder names, and the eng\mgmt.proj did not exist. I did a merge from master repo and will update.

File: src/SDKs/maintenance/Management.Maintenance/Microsoft.Azure.Management.Maintenance.csproj (hunk 1 above)
Comment (abkmr) [reply src/SDKs/maintenance/Management.Maintenance/Microsoft.Azure.Management.Maintenance.csproj]: added packageReleaseNotes

**Comment by abkmr** [comment]
> Hi @abkmr , please
> 
> * check in `AssemblyInfo.cs`
> * write tests for SDK code
> 
> and please see my other inline comments. Thanks!

Added tests and AssemblyInfo.cs

File: src/SDKs/_metadata/maintenance_resource-manager.txt (hunk 2 above)
Comment (isra-fel) [reply src/SDKs/_metadata/maintenance_resource-manager.txt]: Could you remove this file then?

**Review by isra-fel — APPROVED** [review]

**Pull request merged**